src/components/SendTransaction/ConfirmTransaction.tsx

```diff
@@ -242,6 +244,8 @@ export const ConfirmTransaction = ({
         </InfoBlock>
       )}
 
+      {formData.isAccountUnsafe && <IsAccountFlagged flagType="unsafe" />}
```

Comment:
show unsafe account message on Confirm Transaction.

We don't bother showing the malicious warning as the user should not be able to get to this screen with a malicious destination address